AppController/djinn.rb

```diff
@@ -4609,17 +4649,17 @@ def starts_new_apps_or_appservers()
 
   # This function compares the applications that should be running with
   # the ones we have setup, and removes the stopped applications.
-  def check_stopped_app()
+  def check_stopped_apps()
     uac = UserAppClient.new(my_node.private_ip, @@secret)
     Djinn.log_debug("Checking applications that have been stopped.")
     begin
       app_list = uac.get_all_apps()
-    rescue FailedNodeException
-      Djinn.log_warn("Failed to get app listing: retrying.")
-      retry
+    rescue FailedNodeException => except
```

Comment:
Sorry, that comment was supposed to be for the block below. I added a new comment down there.